sycl/test/basic_tests/sycl_maximum_minimum.cpp

```diff
@@ -0,0 +1,27 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %t.out
+
+#include <CL/sycl.hpp>
```

Comment:
Let's align with SYCL 2020: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:headers-and-namespaces